modules/luci-mod-system/htdocs/luci-static/resources/view/system/system.js

```diff
@@ -282,6 +283,14 @@ return view.extend({
 			o.ucisection = 'ntp';
 			o.depends('enabled', '1');
 
+			o = s.taboption('timesync', widgets.NetworkSelect, 'interface',
+				_('Bind NTP server to the specified interface or, if unspecified, to all'));
```

Comment:
Please make the following changes so we have a short and a detail description before.
```suggestion
				_('Bind NTP server'),
				_('Provide the NTP server to the selected interface or, if unspecified, to all interfaces'));
```

Reply:
Corrected. Thanks for the advice!